Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMSTR.table.al

```diff
@@ -182,20 +182,27 @@ table 40099 "GP Checkbook MSTR"
     procedure MoveStagingData()
     var
         BankAccount: Record "Bank Account";
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
+        MigrateInactiveCheckbooks: Boolean;
     begin
+        MigrateInactiveCheckbooks := false;
+        if GPCompanyAdditionalSettings.Get(CompanyName()) then
+            MigrateInactiveCheckbooks := GPCompanyAdditionalSettings."Migrate Inactive Checkbooks";
+
         if FindSet() then
             repeat
-                if not BankAccount.Get(CHEKBKID) then begin
-                    BankAccount.Init();
-                    BankAccount."No." := DelChr(CHEKBKID, '>', ' ');
-                    BankAccount.Name := DelChr(DSCRIPTN, '>', ' ');
-                    BankAccount."Bank Account No." := DelChr(BNKACTNM, '>', ' ');
-                    BankAccount."Last Check No." := GetLastCheckNumber(NXTCHNUM);
-                    BankAccount."Balance Last Statement" := Last_Reconciled_Balance;
-                    BankAccount."Bank Acc. Posting Group" := GetBankAccPostingGroup(ACTINDX);
-                    UpdateBankInfo(DelChr(BANKID, '>', ' '), BankAccount);
-                    BankAccount.Insert(true);
-                end;
+                if not BankAccount.Get(CHEKBKID) then
+                    if MigrateInactiveCheckbooks or not INACTIVE then begin
+                        BankAccount.Init();
+                        BankAccount."No." := DelChr(CHEKBKID, '>', ' ');
```

Comment:
Minor - we have new constructs for deleting, replacing and other string operations. Feel free to wontfix this one, it is more FYI

Reply:
Thanks for the information. Is there any documentation on this that you can link to?

Reply:
@jaymckinney-enavate 
I think @nikolakukrika is referin to the new string operations that were added to AL, e.g. TrimEnd (https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/text/text-trimend-method)

Reply:
we will look to update the entire extension with the new constructs in a later PR.